exchange/gekkoBroker.js

```diff
@@ -0,0 +1,134 @@
+/*
+  The broker manages all communicatinn with the exchange, delegating:
+
+  - the management of the portfolio to the portfolioManager
+  - the management of actual trades to orders.
+*/
+
+const _ = require('lodash');
+const async = require('async');
+const events = require('events');
+const moment = require('moment');
+const checker = require('./exchangeChecker');
+const errors = require('./exchangeErrors');
+const Portfolio = require('./portfolioManager');
+// const Market = require('./market');
```

Comment:
Is there WIP going on here?

Reply:
early wip, this line is not ready for prime time just yet ;) I should remove..